api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/EriDSDCohortQueries.java

```diff
@@ -715,7 +712,7 @@ private CohortDefinition getPatientsWhoCompletedRapidFlow() {
     CodedObsCohortDefinition cd = new CodedObsCohortDefinition();
     cd.addParameter(new Parameter("onOrBefore", "Before Date", Date.class));
     cd.addParameter(new Parameter("onOrAfter", "After Date", Date.class));
-    cd.addParameter(new Parameter("locationList", "Location", Location.class));
+    cd.addParameter(new Parameter("location", "Location", Location.class));
```

Comment:
@simaomazive why did you change this?

Reply:
I made these changes in order to be able to run livetst for N9. But I will put it back. I forgot